### PR TITLE
Allow `params` keyword completion in lambdas

### PR DESCRIPTION
**Pull request by jjonescz**
Relates to test plan #62485.

### CODE REVIEW

File: src/Features/CSharp/Portable/Completion/KeywordRecommenders/ParamsKeywordRecommender.cs [hunk 1]
```diff
@@ -15,6 +15,10 @@ public ParamsKeywordRecommender()
         }
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
-            => context.SyntaxTree.IsParamsModifierContext(context.Position, context.LeftToken);
+        {
+            var syntaxTree = context.SyntaxTree;
+            return syntaxTree.IsParamsModifierContext(context.Position, context.LeftToken) ||
+                syntaxTree.IsPossibleLambdaParameterModifierContext(position, context.LeftToken, cancellationToken);
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/ParamsKeywordRecommender.cs:21]: We should fix up IsParamsModifierContext to support lambdas.

Comment (jjonescz) [reply src/Features/CSharp/Portable/Completion/KeywordRecommenders/ParamsKeywordRecommender.cs]: Right, that's better, thanks. Updated: 4c2f831fdf74362489629e3ef36f5d09a5bfd804

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
LGTM.  thanks!

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**